docs/config/enabling-agent-service.rst

```diff
@@ -0,0 +1,19 @@
+==================================
+Enabling the Flocker agent service
+==================================
+
+On CentOS 7
```

Comment:
Make the titles just the OS name - the "On" part doesn't help when scanning for my Operating System.